### PR TITLE
Use pathlib.Path in the code

### PR DESCRIPTION
**Pull request by eumiro**
This PR is independent from #86 as it adds `pathlib.Path` to the codebase and not to tests.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,24 +1,20 @@
 #!/usr/bin/env python
 
 import re
-from setuptools import find_packages, setup
-
+from pathlib import Path
 
-def read(path):
```
Comment (Lucas-C) [inline setup.py:7]: 👍

File: fpdf/fpdf.py [hunk 2]
```diff
@@ -544,20 +547,16 @@ def add_font(self, family, style="", fname=None, uni=False):
         if fontkey in self.fonts:
             return
         if uni:
-            if os.path.exists(fname):
-                ttffilename = fname
-            elif FPDF_FONT_DIR and os.path.exists(os.path.join(FPDF_FONT_DIR, fname)):
-                ttffilename = os.path.join(FPDF_FONT_DIR, fname)
-            elif SYSTEM_TTFONTS and os.path.exists(os.path.join(SYSTEM_TTFONTS, fname)):
-                ttffilename = os.path.join(SYSTEM_TTFONTS, fname)
+            for parent in (".", FPDF_FONT_DIR, SYSTEM_TTFONTS):
```
Comment (Lucas-C) [inline fpdf/fpdf.py:550]: 👍

File: fpdf/fpdf.py [hunk 3]
```diff
@@ -47,10 +48,11 @@
 )
 
 LOGGER = logging.getLogger(__name__)
+HERE = Path(__file__).resolve().parent
```
Comment (Lucas-C) [inline fpdf/fpdf.py:51]: `PARENT_DIR` could be more explicit than just `HERE`

**Review by Lucas-C — APPROVED** [review]

File: fpdf/fpdf.py (hunk 3 above)
Comment (eumiro) [reply fpdf/fpdf.py:51]: It is `PARENT` or `DIR`, but `PARENT_DIR` sounds double level to me. I personally use and prefer the `HERE` name.

**Pull request merged**